src/main/java/org/opensearch/ad/model/AnomalyResult.java

```diff
@@ -464,10 +483,25 @@ public String getError() {
         return error;
     }
 
-    public List<Entity> getEntity() {
+    public Entity getEntity() {
         return entity;
     }
 
+    public String getModelId() {
+        return modelId;
+    }
+
+    /**
+     * Anomaly result index consists of overwhelmingly (99.5%) zero-grade non-error documents.
+     * This function exclude the majority case.
+     * @return whether the anomaly result is important when the anomaly grade is not 0
+     * or error is there.
+     */
+    public boolean isHighPriority() {
+        // AnomalyResult.toXContent won't record Double.NaN and thus make it null
+        return (getAnomalyGrade() != null && getAnomalyGrade() > 0) || getError() != null;
```

Comment:
Why isnt anomalygrade initialized to 0?

Reply:
anomalygrade = 0 means there is no anomaly.  We want to differentiate no anomaly and don't know case.